pkgs/by-name/fl/flattenReferencesGraph/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  callPackage,
+  nix-gitignore,
+  python3Packages,
+}:
+let
+  helpers = callPackage ./helpers.nix { };
+  pythonPackages = python3Packages;
+
+in
+pythonPackages.buildPythonApplication {
+  version = "0.1.0";
+  pname = "flatten-references-graph";
+
+  # Note: this uses only ./src/.gitignore
+  src = nix-gitignore.gitignoreSource [ ] ./src;
```

Comment:
Perhaps this could be a `lib.fileset.toSource` picking only the needed files.

Reply:
done in 9ae0f7757760

Reply:
looks like this change is causing a [ci check](https://github.com/NixOS/nixpkgs/actions/runs/12091053669/job/33718955875?pr=122608) to fail with:

```
- pkgs/by-name/fl/flattenReferencesGraph: File package.nix at line 18 contains the path expression "./src/__pycache__" which cannot be resolved: No such file or directory (os error 2).
```

I guess I can just use a union there, and if any new entries are added to src, the author will need to remember to add them to the union

Reply:
fixed in ba5a5fac7bb68edfb1de329ca037ebe0d4094560